### DISCUSSION TITLE
Boom beds

### DISCUSSION THREAD

**Discussion by emielderckx** [PurpurMC/Purpur#444] [__Completed__] [answered]
I don't really know if this is a plugin thingy but I think its cool for Purpur.

**The idea is to have 2 per world settings:**
One that let's you choose if a bed explodes if a player sleeps in it.

And an other setting that let's you choose if a bed explodes if a villager sleeps in it. (my people are bringing villagers to the end and I want to destroy them all)

**Comment by TetraTheta** [comment]
Since there is `PlayerBedEnterEvent` in bukkit, `PlayerBedFailEnterEvent` in paper, I think you can do player thing with plugin.  
But there is no event for villager-bed related event in Purpur, that would require PR for that event.

**Comment by craftish37** [comment]
The boom boom villagers is a good idea, and I think it’s not even possible with plugins (maybe ?)

**Reply by TheAwesome98-Real** [reply]
> The boom boom villagers is a good idea, and I think it’s not even possible with plugins (maybe ?)

It is possible.

**Comment by TheAwesome98-Real** [comment]
> my people are bringing villagers to the end and I want to destroy them all

`/kill @e[type=minecraft:villager]`?
also, what's so bad about end villagers?

**Reply by emielderckx** [reply]
> > my people are bringing villagers to the end and I want to destroy them all
> 
> `/kill @e[type=minecraft:villager]`?
> also, what's so bad about end villagers?

I'm an evil man and I just want to destroy those pesky villagers. 
(they cause lag and this is such a fun way to kill most of them in all worlds)

**Reply by TheAwesome98-Real** [reply]
> > > my people are bringing villagers to the end and I want to destroy them all
> > 
> > 
> > `/kill @e[type=minecraft:villager]`?
> > also, what's so bad about end villagers?
> 
> I'm an evil man and I just want to destroy those pesky villagers.
> (they cause lag and this is such a fun way to kill most of them in all worlds)

lol

**Comment by katekerllenevich** [comment, marked as answer]
Added in b152230
